spec/longest_word_spec.rb

```diff
@@ -5,4 +5,8 @@
   it 'should return correct results' do
     expect(longest_word("Here is the longest word")).to eq "longest"
   end
+
+  it 'should return last longest word' do
```

Comment:
we don't need the `should` here.

`it 'returns the last longest word'` here is enough